content-based-message-routing/README.md

```diff
@@ -0,0 +1,476 @@
+# Content Based Message Routing
+
+## Overview
+
+Let's develop a service that accepts requests to make an appointment at a hospital and makes a call to a particular hospital endpoint depending on the content provided.
+
+To implement this use case, you will develop a REST service with a single resource using Visual Studio Code with the Ballerina Swan Lake extension. The resource will receive the user request, select the hospital endpoint based on the `hospital_id`, send a request to the hospital service and respond with the correct reservation details
```

Comment:
hospital endpoint vs hospital service. Let's ensure the flow with the same terms where applicable. 

```suggestion
To implement this use case, you will develop a REST service with a single resource using Visual Studio Code with the Ballerina Swan Lake extension. The resource will receive the user request, select the hospital endpoint based on the `hospital_id`, send a request to the hospital service, and respond with the correct reservation details
```